dbms/src/Server/Compiler-5.0.0/CMakeLists.txt

```diff
@@ -51,3 +51,8 @@ libtinfo.a
 
 PUBLIC ${ZLIB_LIBRARIES} ${EXECINFO_LIBRARY} Threads::Threads
 )
+
+if (MAKE_STATIC_LIBRARIES)
+    # fix strange static error: undefined reference to 'std::error_category::~error_category()'
+    target_link_libraries(clickhouse-compiler-lib PUBLIC stdc++)
```

Comment:
This fix nedded only for old distributions (ubuntu trusty, xenial)
In ubuntu artful anything ok.
ant this fix adds dependency to shared system lib - it unacceptable for our fully static package.